.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: php
 
 php:
-  - 5.5
-  - 5.6
-  - 7.0
+  - '5.5'
```

Comment:
I have never seen it defined as strings before in any repo.
Can you please clarify why you think this is an improvement?
The PR should be limited to only the absolutly required changes to make it easier to be approved and passed, right now there seems to be quite some noise changes on top that do not seem clear without further explanations IMO.